data.sql

```diff
@@ -0,0 +1,14 @@
+
+/* store in database sample data. */
+
+INSERT INTO animals (name) VALUES ('Luna');
+INSERT INTO animals (name) VALUES ('Daisy');
+INSERT INTO animals (name) VALUES ('Charlie');
```

Comment:
[OPTIONAL]
- I assume that these were present when the files were generated. Feel free to remove them as you do not need them. 😃